src/components/posts/PostItem.js

```diff
@@ -28,16 +35,28 @@ function PostItem(props) {
           </div>
           <h5>{props.content}</h5>
         </div>
+        <div className={classes.likeCount}>{likeCount}</div>
         <div className={classes.actions}>
           <div className={classes.like}>
-            
-            <button onClick={likeHandler}>like</button>
+            <LikeButton
+              id={props.id}
+              setLikeCount={setLikeCount}
+              likeCount={likeCount}
+            />
           </div>
           <button onClick={editHandler}>edit</button>
-          {/* <button onClick={deleteHandler}>delete</button> */}
-          <DeleteButton id={props.id} setRefreshPost={props.setRefreshPost}/>
+          <DeleteButton id={props.id} setRefreshPost={props.setRefreshPost} />
         </div>
       </Post>
+      {showEdit && (
```

Comment:
Nice use of `&&` to limit what is being shown

Reply:
thanks